.github/workflows/test-older-node.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [12, 14, 16]
+        node: [14, 16]
```

Comment:
Should this include 18 now?

Reply:
We test `18` and lts in `.github/workflows/test.yml`.

I think the main reason we have two workflows is because Stylelint doesn't support anything older than `18`.

The `test.yml` does the `json` file test and a Stylelint check on the site.
The `test-older-node.yml` only does the `json` file test.

Reply:
Ahh got it! Makes sense!